packages/core/src/banner/BannerContent.tsx

```diff
@@ -1,10 +1,13 @@
-import { ReactNode } from "react";
+import { ComponentPropsWithoutRef, ReactNode } from "react";
 import { makePrefixer } from "../utils";
 
 import "./BannerContent.css";
+import { clsx } from "clsx";
 
 const withBaseName = makePrefixer("saltBannerContent");
 
-export const BannerContent = ({ children }: { children: ReactNode }) => {
-  return <span className={withBaseName()}>{children}</span>;
+export interface BannerContentProps extends ComponentPropsWithoutRef<"span"> {}
+
+export const BannerContent = ({ className, ...rest }: BannerContentProps) => {
```

Comment:
so what would be the use of the BannerContent if it's the only one left on composition?

Reply:
It ensures alignment at the moment and it current serves as a place to put the role. Although I would have to check with Stella

Reply:
the role can be placed directly on Banner. I believe this can be merged with my changes on https://github.com/jpmorganchase/salt-ds/pull/1643 which includes removal of BannerContent

Reply:
It could be placed but then the button would be announced which might not be desirable